src/js/utils.js

```diff
@@ -13,7 +13,12 @@ exports.parseQueryParams = () => {
   const items = search.slice(1).split('&');
   return items.reduce((acc, item) => {
     const [key, value] = item.split('=');
-    acc[decodeURIComponent(key)] = decodeURIComponent(value).split(',');
+    const val = decodeURIComponent(value).split(',');
+    if (!acc[decodeURIComponent(key)]) {
```

Comment:
nit: I'd extract that decoded key into its own var

Reply:
Good call - done!